schemas/WebpackOptions.json

```
@@ -983,8 +983,16 @@
       "properties": {
         "assetModuleFilename": {
           "description": "The filename of asset modules as relative path inside the `output.path` directory.",
-          "type": "string",
-          "absolutePath": false
+          "anyOf": [
+            {
+              "type": "string",
+              "absolutePath": false
+            },
+            {
+              "instanceof": "Function",
+              "tsType": "Function"
```

This could be more exact.
```suggestion
              "tsType": "((pathData: import(\"../lib/Compilation\").PathData) => string)"
```